src/layout/MasterLayout.cpp

```diff
@@ -153,28 +153,28 @@ void CHyprMasterLayout::onWindowRemovedTiling(CWindow* pWindow) {
     if (!PNODE)
         return;
 
+    const auto WORKSPACEID = PNODE->workspaceID;
+    const auto MASTERSLEFT = getMastersOnWorkspace(WORKSPACEID);
+    const bool SMALL_SPLIT = g_pConfigManager->getConfigValuePtr("master:allow_small_split")->intValue == 1;
```

Comment:
nuh-uh.

config value ptr has to be static

no snake case

Reply:
sorry, not super familliar with C++.
did you need me to create the SMALL_SPLIT const as a pointer rather than copying the value?
happy to change SMALL_SPLIT to SMALLSPLIT

Reply:
changes pushed as i understand them.
let me know if anything else needs work